src/20240720_AaveV3EthereumLido_LidoEthereumInstanceActivation/LidoEthereumInstanceActivation.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Lido Ethereum Instance Activation"
+author: "Catapulta @catapulta_sh"
+discussions: "https://governance.aave.com/t/arfc-deploy-a-lido-aave-v3-instance/18047"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0x9403b7f704ade0d6510591e4846fd85c84b19d134c0814511af914751ecfad5d"
+---
+
+## Simple Summary
+
+## Motivation
+
+## Specification
+
+The table below illustrates the configured risk parameters for **wstETH**
+
+| Parameter                          |                                      Value |
```

Comment:
@kartojal imo better to club it in a single table

Reply:
is generated automatically

Reply:
looks way better and compact, added table with both assets